bokeh/models/math_text.py

```diff
@@ -0,0 +1,18 @@
+# Bokeh imports
+from ..core.properties import Nullable, String
+from ..model import Model
+
+
+class MathText(Model):
+    """
+    Class for signaling you want you text converted to LaTex
```

Comment:
`LaTeX` != `TeX` != `TeX` math. Besides this, you're not converting text to anything, but you're rendering `TeX` math formatted strings. If anything this is a marker indicating that an `str` value should be interpreted as `TeX` math, instead of a raw string, the same as we may have a `HTML` marker that indicates HTML content in a string instead of raw string (resulting in using e.g. `innerHTML` instead of `textContent` later on).